.github/workflows/ci.yaml

```diff
@@ -137,6 +137,8 @@ jobs:
       # Note: ${{ github.sha }} not working, see https://github.com/actions/checkout/issues/299
       NEURO_EXTRAS_IMAGE: ghcr.io/neuro-inc/neuro-extras:${{ github.event.pull_request.head.sha
         || github.sha }}
+      # https://github.com/google-github-actions/setup-gcloud/issues/381
+      CLOUDSDK_PYTHON: python3.8
```

Comment:
I suggest passing required vars for each step, not globally for the task.
It can help with avoiding security wholes like introduced by codecov upload script last year.
Leaking tokens doesn't sound good.

Reply:
Changed it in 1a52540, pls check it out now.

Reply:
I mean removing all job-global envs above also.

Reply:
done in 
e672b56